src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBadGenomicKmersSpark.java

```diff
@@ -31,7 +32,9 @@
     private static final long serialVersionUID = 1L;
     @VisibleForTesting static final Long MAX_KMER_FREQ = 3L;
     private static final int REF_RECORD_LEN = 10000;
-    private static final int CHUNK_SIZE = 3000000;
+    // to give a task size of 100Kb -- the max size recommended in the Spark documentation
+    private static final int REF_RECORDS_PER_PARTITION = 102400 / REF_RECORD_LEN;
```

Comment:
I agree that we can't just assume the presence of massive amounts of memory, and that we need the functionality in https://github.com/broadinstitute/gatk/issues/1456 to scale things back in low-memory environments, but I do think by default it makes sense in our default settings to optimize for the large-memory case, and the default values we choose for things like partition size should not cripple performance given a beefy cluster.